shared/src/main/scala/coop/rchain/store/KeyValueStoreSyntax.scala

```diff
@@ -8,7 +8,9 @@ import cats.syntax.all._
 import scodec.Codec
 
 trait KeyValueStoreSyntax {
-  implicit final def sharedSyntaxKeyValueStore[F[_]](store: KeyValueStore[F]): KeyValueStoreOps[F] =
+  implicit final def sharedSyntaxKeyValueStore[F[_]](
+      store: KeyValueStore[F]
+  ): KeyValueStoreOps[F] =
```

Comment:
It seems this code is not changed.

Reply:
Yes. It seems my formatter format itself.